src/Service.GraphQLBuilder/GraphQLTypes/SupportedTypes.cs

```diff
@@ -8,7 +8,7 @@ namespace Azure.DataApiBuilder.Service.GraphQLBuilder.GraphQLTypes
     /// The type names mentioned here are Hotchocolate scalar built in types.
     /// The corresponding SQL type name may be different for e.g. UUID maps to Guid as the SQL type.
     /// </summary>
-    public static class SupportedTypes
+    public static class SupportedHotChocolateTypes
```

Comment:
Updated filename to reflect this name would be beneficial for discoverability. With that being said, do you think it would be worth splitting these two classes into separate files?

Reply:
I think `SupportedTypes` make sense at the moment. If the number of classes get too big, we can seperate them into different files.